test/tests/outputs/pp_as_reporter/pp_as_reporter.i

```diff
@@ -0,0 +1,24 @@
+[Mesh/gen]
+  type = GeneratedMeshGenerator
+  dim = 1
+[]
+
+[Problem]
+  solve = false
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Postprocessors/data]
+  type = FunctionValuePostprocessor
+  function = 1980
+[]
+
+[Outputs]
+  [out]
+    type = JSON
+    postprocessors_as_reporters = true
```

Comment:
I think you need to add `execute_system_information_on = none` or else it will compare git versions, right?

Reply:
I skip those keys in the tests specifications

Reply:
skip_keys = 'current_time executable'

Reply:
but I need to git version as well

Reply:
I just changed it to your method, it is simpler.